src/Components/Header/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import './Header.css';
+
+const Header = () => {
+
+  return (
+    <div className="header">
+      <div className="title-container">
+        <header className="header-title"><Link to="/" style={{textDecoration: 'none'}}>Fantasy Fellowship</Link></header>
+      </div>
+      <div className="nav-container">
+        <Link to="/fellowship" style={{textDecoration: 'none'}}>Your Fellowship</Link>
```

Comment:
Same question here. I think in tomatillos I ran into this also, trying to change the default styling on Link. I didn't really want to add a className to every single link, but css doesn't accept Link as an element. BUT, I learned that under the hood Link is just an html <a> tag. So if you want your links not to be underlined, you can stick a css rule for <a> wherever you want. I think I put it in App, so it just applied everywhere. 
a {
  text-decoration: none;
}